coderedcms/project_template/sass/website/migrations/0001_initial.py

```diff
@@ -5,7 +5,7 @@
  """
 {% endcomment %}
 {% verbatim %}
-# Generated by Django 2.2.4 on 2019-08-03 16:14
+# Generated by Django 2.2.4 on 2019-08-12 14:12
```

Comment:
This file should be identical to `project_template/basic/website/migrations/0001_initial.py`. The same migrations from basic must also be copied to sass.